tasks/scripts.js

```diff
@@ -10,19 +11,14 @@ module.exports = (gulp, $, pkg) => {
     if (!pkg.gulpPaths.scripts.src) { return false }
     const options = getOptions($, pkg.gulpPaths.scripts.options, opts);
 
-    let stream = gulp.src(pkg.gulpPaths.scripts.src, { sourcemaps: options.sourcemaps })
+    return gulp
+      .src(pkg.gulpPaths.scripts.src)
       .pipe($.if(!options['fail-after-error'], $.plumber()))
-      .pipe($.babel({
-        presets: [['@babel/preset-env', { modules: false }]],
-      }))
-      .pipe($.if(options.concat, $.concat(pkg.title.toLowerCase().replace(/[^a-z]/g,'') + '.js')))
-      .pipe(gulp.dest(pkg.gulpPaths.scripts.dest, { sourcemaps: options.sourcemaps }))
```

Comment:
Depending on the configuration in PHP, the application will either load source or minified files, which can be changed at runtime.  This is why the compilation process writes out both the minified/optimized `.min.js` file as well as the only concatenated source `.js` file.  Can we retain that?

Reply:
Will check the script to keep this functionality.

Reply:
Already changed this. Now the script generates both files. It also was necessary to add an extra script to maintain the folder structure as it was generated before.